src/components/reports/formSteps/Sample.tsx

```diff
@@ -16,8 +16,8 @@ const Sample: FC<Props> = (props) => {
       <h2>Sample</h2>
       <FieldSet name="sample.specimenCode" label="Barcode" />
       <FieldSet name="sample.specimenType" label="Specimen type" />
-      <FieldSet type="date" name="sample.collectionDateTime" label="Sample collected date" />
-      <FieldSet type="date" name="sample.receivedDateTime" label="Sample received date" />
+      <FieldSet type="datetime" name="sample.collectionDateTime" label="Sample collected date" />
+      <FieldSet type="datetime" name="sample.receivedDateTime" label="Sample received date" />
```

Comment:
Is the full date time strictly needed here? If so, we should give the user guidance on the format here, possibly with a placeholder date. 

There should also be an html element available that lets users enter the full date and time, perhaps we could make a component to handle this? 

We will probably experience similar warnings to the other date fields (see comment above), but perhaps that is something we can tackle in a further PR?

Reply:
Yeah for these ones it is datetime that they have, waiting for clarification on this but I think datetime specifically. Agreed that this ties in with the date above